students/rmart300/lesson09/donor_collection.py

```diff
@@ -0,0 +1,92 @@
+import os
+from donor import Donor
+
+class DonorCollection(object):
+
+    letter_directory = 'temp/'
+
+    def __init__(self):
+        # dictionary to hold donors and list of donation amounts
+        donor_list = ['Fred Smith', 'Terrie Ann', 'Murray Martin', 'Josh Jones', 'Jane Doe']
```

Comment:
I'd recommend letting the user create this list when the object is instantiated, rather than beginning with the same hard-coded list for each DonorCollection object.